content/integrations/azure.md

```diff
@@ -18,25 +18,73 @@ sidebar:
+12. Scroll down to Keys and create a new key (1 or 2 years are both acceptable)
+	- Note: the client secret key will be shown after you save
+13. Scroll down to "Permissions To Other Applications"
+14. Click Add Application
+15. Select Windows Azure Service Management API and press the check mark on the bottom right of the screen
+16. Under Delegated Permissions, check off “Access Azure Service Management as organization users (preview)” <img src="/static/images/azure/app_permissions_dropdown.png" style="width: 50%;display:block;"/>
+17. Click "Save" at the bottom of the screen 
+18. Scroll back up to Keys, and copy and paste the generated Client Secret key in the form on the Datadog setup page <img src="/static/images/azure/client_secret_form.png" style="width: 50%;display:block;display:block;"/>
+
+#### Giving Read Permissions to the Application
+
+1. Navigate to [portal.azure.com][2]
+2. Navigate to “Subscriptions” on the left hand menu <img src="/static/images/azure/subscriptions_icon.png" style="width: 25%;display:block;"/>
+3. Click on the subscription you would like to monitor
+4. Click on the user icon <img src="/static/images/azure/user_button.png" style="width: 50%;display:block;"/>
+5. Click "Add" User <img src="/static/images/azure/add_user_button.png" style="width: 25%;display:block;"/>
+6. Select “Reader” as a role <img src="/static/images/azure/reader_icon.png" style="width: 25%;display:block;"/>
+7. Search/select for the name of the Application you just created (i.e. Datadog Auth App)
+8. Click Select
+9. Press OK
+10. Repeat this process for any other subscriptions you would like to monitor
+
+**Navigate back to the Datadog setup page, and click "Update Configuration" to complete the installation process**
+
+**In a few minutes, metrics from applications under that subscription will begin to appear!**
+
 
-To setup Azure monitoring, go to [Azure integrations][1] and follow instructions on the page.
 
 ![Azure default dashboard](/static/images/azure.png)
 
 Learn more about how to monitor Azure performance metrics thanks to [our series of posts](https://www.datadoghq.com/blog/how-to-monitor-microsoft-azure-vms/). We detail the key performance metrics, how to collect them, and how to use Datadog to monitor Azure VMs.
 
 #### Enable diagnostics
-To enable agent-less monitoring, you must enable diagnostics. Right now this is only support by Windows based machines. To do this, first go to [Azure preview portal][2] then follow the instructions below.
+To collect advanced metrics for virtual machines, you must enable diagnostics. Right now this is only support by Windows based machines. To do this, first go to [Azure Portal][2] then follow the instructions below.
 ![](/static/images/azure_diag_manual.png)
 
 After locating your VM:
+
 1. Click on the CPU percentage today panel to show metrics panel
 2. Click on Diagnostics
 3. Shift the switch to open
```

Comment:
Can we provide links to information on automated steps for enabling this during provisioning? or via the api? Doing this one by one on hundreds or thousands of machines doesn't seem scale-able.